release.nix

```diff
@@ -3,12 +3,12 @@
     iosSdkVersion = "10.2";
   }
 , local-self ? import ./. self-args
+, cacheBuildSystems ? [ "x86_64-linux" "x86_64-darwin" ]
```

Comment:
We should at some point rename these `supportedPlatforms` for consistency with Nixpkgs.

Reply:
`supportedSystems`